docs/docs/getting-started/installation/aws.md

```diff
@@ -244,47 +244,40 @@ airy create --provider aws --init-only
 Then edit your `airy.yaml` file and add the following configuration
 
 ```sh
-kubernetes:
-  host: myairy.myhostname.com
 ingress:
+  host: myairy.myhostname.com
   https: true
   letsencryptEmail: "mymail@myhostname.com"
 
 ```
 
-The `kubernets.host` value should be set to your desired hostname. Configure the e-mail address you want to use for your Let's Encrypt registration under `ingress.letsencryptEmail`.
+The `ingress.host` value should be set to your desired hostname. Configure the e-mail address you want to use for your Let's Encrypt registration under `ingress.letsencryptEmail`.
 
 After setting these parameters, create your `Airy Core` instance with the following option:
 
 ```sh
-airy create --provider aws --provider-config hostUpdate=false
+airy create --provider aws
 ```
 
 :::note
-In case you have created your Airy Core instance without Let's Encrypt and want to add it later, you must use the `airy upgrade` command.
-
-Even if you don't upgrade to a new version, just modify your `airy.yaml` file as explained in this section and run
-
-`airy upgrade`
-
-After the upgrade is done, continue with setting up your DNS and starting the ingress controller.
+In case you have created your Airy Core instance without Let's Encrypt and want to add it later, modify your `airy.yaml` file accordingly and continue with the process from the next section. 
 :::
 
 #### Setup your DNS
 
-You should create a CNAME DNS record for the hostname that you set under `kubernetes.host` in the previous step to point to the hostname of the LoadBalancer, created by AWS for the ingress service:
+You should create a CNAME DNS record for the hostname that you set under `ingress.host` in the previous step to point to the hostname of the LoadBalancer, created by AWS for the ingress service:
 
 ```sh
 export KUBECONFIG="PATH/TO/DIR/kube.conf"
-kubectl get --namespace kube-system service traefik --output jsonpath='{.status.loadBalancer.ingress[0].hostname}{"\n"}'
+kubectl get --namespace kube-system service ingress-nginx-controller --output jsonpath='{.status.loadBalancer.ingress[0].hostname}{"\n"}'
 ```
 
-#### Start the ingress controller
+#### Run airy upgrade
 
-If the ingress controller is started before the DNS record is added, the Let's Encrypt servers will block and throttle the registration attempts. That is why we recommend starting the ingress controller after the DNS record is added.
+If the ingress controller is started before the DNS record is added, the initial Let's Encrypt requests will fail and then all the following registration attempts will be blocked and throttled. That is why the generation of the Let's Encrypt certificates is disabled by default. In order to complete the setup, you must run the upgrade command.
 
 ```sh
-kubectl -n kube-system scale statefulset -l k8s-app=traefik-ingress-lb --replicas=1
+airy upgrade
```

Comment:
The installation of the `ClusterIssuer` resource is done only on `upgrade` and not on `install`, so that we leave time for the user to create his DNS record, after the creation of the cluster.